### DISCUSSION TITLE
Regarding other coins?

### DISCUSSION THREAD

**Discussion by snaildos** [btcpayserver/btcpayserver#2600] [General]
Is there going to support other coins?
Such as monero? Dogecoin?
Regards,
SnailDOS

**Comment by Kukks** [comment]
Already supported. Please note we do not provide any official support for altcoins. 
https://docs.btcpayserver.org/FAQ/FAQ-Altcoin/

**Reply by Eskyee** [reply]
REMEMBER Altcoin = Shitcoin 
Bitcoin and Shitcoin mix like oil and water 🧐

**Comment by snaildos** [comment]
Alright. Makes sense, thank you. :)
